tokenizers/src/pre_tokenizers/sequence.rs

```diff
@@ -6,19 +6,27 @@ use serde::{Deserialize, Serialize};
 #[derive(Clone, Debug, PartialEq)]
 #[macro_rules_attribute(impl_serde_type!)]
 pub struct Sequence {
-    pretokenizers: Vec<PreTokenizerWrapper>,
+    pre_tokenizers: Vec<PreTokenizerWrapper>,
 }
 
 impl Sequence {
-    pub fn new(pretokenizers: Vec<PreTokenizerWrapper>) -> Self {
-        Self { pretokenizers }
+    pub fn new(pre_tokenizers: Vec<PreTokenizerWrapper>) -> Self {
+        Self { pre_tokenizers }
+    }
+
+    pub fn get_pre_tokenizers(&self) -> &[PreTokenizerWrapper] {
+        &self.pre_tokenizers
+    }
+
+    pub fn get_pre_tokenizers_mut(&mut self) -> &mut [PreTokenizerWrapper] {
```

Comment:
These two new functions are just mimicking the interface you already have for the `Sequence` `Normalizer`.